bundle/Resources/translations/validators.en.xlf

```diff
@@ -36,6 +36,11 @@
         <target>Section identifier may only contain letters from "a" to "z", numbers and underscores.</target>
         <note>key: ez.section.identifier.format</note>
       </trans-unit>
+      <trans-unit id="963a415cd08151ecf8726e651a7a36140e05f4b5" resname="ez.role.identifier.unique">
+        <source>Non unique role identifier "%identifier%"</source>
```

Comment:
Not a sentence. I'd explain it more clearly, with something like: "The role identifier 'blarghhh' is used by another role. You must enter a unique identifier."